website/content/en/preview/AWS/user-data.md

```diff
@@ -13,27 +21,19 @@ kind: AWSNodeTemplate
 metadata:
```

Comment:
Can we refer to one of the examples you've created above for this?

Reply:
The reason I had it here is because I didn't want our readers to have to jump between our documentation on karpenter.sh and our GitHub repo. I've now added a link under this that links to the examples in GitHub repo so they can go there if needed, so I hope that addresses this?

Reply:
We should be able to reference this like we do in the getting-started guides, no? 

https://github.com/aws/karpenter/blob/main/website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md#create-a-cluster